src/components/Editor.jsx

```diff
@@ -73,6 +76,13 @@ class Editor extends React.Component {
     if (openLine) {
       this.emphasizeLine(openLine);
     }
+    if (value && AST.children && filters) {
```

Comment:
That's odd, I seem to remember it working fine before the merge. Will look into that.